exercises/parallel-letter-frequency/HINTS.md

```diff
@@ -0,0 +1,22 @@
+# Parallel Letter Frequency in Rust
+
+Learn more about concurrency in Rust here:
+
+- [Concurrency](https://doc.rust-lang.org/book/concurrency.html)
+
+## Bonus
+
+This exercise also includes a benchmark, with a sequential implementation
+as a baseline. As of this writing, Bencher is unstable and only available on
```

Comment:
I think this Bonus section could describe more what we want the students to do. Right now this only mentions that the benchmark exists, but not why or what students should do with it.

> This exercise also includes a benchmark, with a sequential implementation as a baseline. You can compare your solution to the benchmark, or you can try to surpass the benchmark using concurrent programming techniques.

Or something like that.